ldapadmin/src/main/webapp/console/app/components/stats/stats.es6

```diff
@@ -67,13 +67,42 @@ class StatsController {
             return value
           }
         }
+
       }
     }
+    let el = $element.find('.chartist')
     this.lines = new Chartist[this.type=='bar' ? 'Bar' : 'Line'](
-      $element.find('.chartist')[0], this.parsed, options
+      el[0], this.parsed, options
+    )
+    this.lines.on('draw', (data) => {
+      if(data.type === 'label') {
+        let ydiff = 8
```

Comment:
Can you add a comment about this magic value : 8, 18 ? I don't understand why `ydiff` take 8 or 18 as value.